src/app/pages/component-sidenav/component-sidenav.scss

```
@@ -33,7 +33,7 @@ app-component-sidenav {
 
   .docs-component-viewer-nav-content {
     width: constants.$sidenav-width;
-    height: 100vh;
+    height: 80vh;
```

It's actually the top nav that we're not accounting for, which is `56px`.
```suggestion
    height: calc(100vh - 56px);
```